src/form.js

```diff
@@ -106,13 +113,21 @@ export default class SimpleteForm extends HTMLElement {
 		}
 	}
 
+	onToggle(ev) {
+		let field = this.searchField;
+		field.setAttribute("aria-expanded", ev.detail.expanded);
```

Comment:
* `ev.detail.expanded` is boolean, so you're relying on automatic stringification here, to result in `"true"` or `"false"`? I suppose that's alright, though being explicit (e.g. `!!expanded.toString()`) wouldn't hurt and might improve readability?
* should this perhaps be called `isExpanded`? (though I suppose there's conflicting precedent with `SimpleteForm#navigating` at least)